HelpSource/Reference/Functions.schelp

```diff
@@ -53,52 +71,82 @@ code::{ |x = 1| ... } :: link::#[1]::
 Pipe style, default value is an expression:
 code::{ |x = (10.rand)| ... } :: link::#[2]::
 
-If the last argument in the list is preceded by three dots (an ellipsis), then all the remaining arguments that were passed will be assigned to that variable as an link::Classes/Array::. Arguments must be separated by commas.
 
-examples:
+In general arguments may be initialized to literals or expressions, but in the case of Function:play or SynthDef:play, they may only be initialized to literals.
```

Comment:
general arguments may be initialized to literals or expressions, but in the case of link::Classes/Function#-play:: or link::Classes/SynthDef#-play::, they may only be initialized to literals.

Reply:
Oh, sorry it was copy-paste mistake.

The correct sentence would be:
>In general arguments may be initialized to literals or expressions, but in the case of link::Classes/Function#-play:: or link::Classes/SynthDef#-play::, they may only be initialized to literals.

Reply:
Please, be careful not to add more work. Your comments are useful, but if the respondent has to invest time to double-check, it is lost again. The best here would be to only suggest the part that is changed, thus ` link::Classes/Function#-play:: or link::Classes/SynthDef#-play::`. Thank you @prko !